test-cases/load/background-reads-overload.yaml

```diff
@@ -0,0 +1,18 @@
+test_duration: 30
+
+prepare_write_cmd: "cassandra-stress user profile=/tmp/cs_profile_background_reads_overload.yaml ops'(insert=10)' no-warmup cl=ONE -port jmx=6868 -mode cql3 native -rate threads=1000 -errors ignore"
+stress_cmd_r: "cassandra-stress user profile=/tmp/cs_profile_background_reads_overload.yaml ops'(simple1=100)' no-warmup cl=ONE duration=20m -port jmx=6868 -mode cql3 native -rate threads=1000 -errors ignore"
```

Comment:
Do we really need to run it for 20 mins?

Reply:
i think we could shorten it a bit, but i still not sure about it, because it doesn't succeed on each run, so i'm not sure how much less we could run

Reply:
yet

Reply:
decreased it to 10 minutes